Assets/__Scripts/TaskExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace __Scripts
```

Comment:
None of the files (at least in __Scripts) should have namespace afaik. Not even sure this is the right namespace.